services/QuillLMS/client/app/bundles/PremiumHub/components/usage_snapshots/snapshotRanking.tsx

```diff
@@ -112,17 +112,24 @@ const SnapshotRanking = ({ label, queryKey, headers, searchCount, selectedGrades
     })
   }
 
+  function selectionsEqual(firstSelection, secondSelection) {
```

Comment:
We should maybe move this function out to the `shared` file in this folder since the code is duplicated.